features/atcryptoauth/README.md

```diff
@@ -1,4 +1,16 @@
 # Atmel Crypto Auth (ATCAECC508A) driver
+### Table of Contents
```

Comment:
Markdown rendering tools should generate a table of contents, you shouldn't write one manually.

Reply:
I thought so as well. But Github doesn't do that.